src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -2920,11 +2920,9 @@ public PreSubmitCallbackHandler<AsylumCase> requestFeeRemissionNotificationHandl
     ) {
 
         return new NotificationHandler(
-            (callbackStage, callback) -> {
-
-                return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                       && callback.getEvent() == Event.REQUEST_FEE_REMISSION;
-            },
+            (callbackStage, callback) -> callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+                   && !isAipJourney(callback.getCaseDetails().getCaseData())
```

Comment:
I just want to double check is that for AipJourney or for ! AipJournet?

Reply:
My journey is AiP, this one(legalRep) is triggers as well on this Event, so I added protection for non-AiP only.
In my journey Im not sending any notifications.

Reply:
ok, I got you